e2e/process-services/start-process-component.e2e.ts

```diff
@@ -60,402 +72,485 @@ describe('Start Process Component', () => {
-            await processServicesPage.goToApp('Task App');
-            await processServiceTabBarPage.clickProcessButton();
-            await processFiltersPage.clickCreateProcessButton();
-            await processFiltersPage.clickNewProcessDropdown();
-            await startProcessPage.checkNoProcessMessage();
-        });
-    });
-
-    describe(' Once logged with user with app', () => {
+    describe('Provider: BPM', () => {
 
         beforeAll(async () => {
-            await loginPage.loginToProcessServicesUsingUserModel(secondProcUserModel);
-        });
-
-        beforeEach(async () => {
-            await navigationBarPage.navigateToProcessServicesPage();
-            await processServicesPage.checkApsContainer();
-        });
-
-        it('[C260441] Should display start process form and default name when creating a new process', async () => {
-            await processServicesPage.goToApp('Task App');
-            await processServiceTabBarPage.clickProcessButton();
-            await processFiltersPage.clickCreateProcessButton();
-            await processFiltersPage.clickNewProcessDropdown();
-            await expect(await startProcessPage.getDefaultName()).toEqual('My Default Name');
-        });
-
-        it('[C260445] Should require process definition and be possible to click cancel button', async () => {
-            await processServicesPage.goToApp('Task App');
-            await processServiceTabBarPage.clickProcessButton();
-            await processFiltersPage.clickCreateProcessButton();
-            await processFiltersPage.clickNewProcessDropdown();
-            await startProcessPage.enterProcessName('');
-            await browser.actions().sendKeys('v\b\b').perform(); // clear doesn't trigger the validator
-            await startProcessPage.checkStartProcessButtonIsDisabled();
-            await startProcessPage.clickCancelProcessButton();
-            await processFiltersPage.checkNoContentMessage();
-        });
-
-        it('[C260444] Should require process name', async () => {
-            await processServicesPage.goToApp(app.title);
-
-            await processServiceTabBarPage.clickProcessButton();
-
-            await processFiltersPage.clickCreateProcessButton();
-            await processFiltersPage.clickNewProcessDropdown();
+            this.alfrescoJsApi = new AlfrescoApi({
+                provider: 'BPM',
+                hostBpm: browser.params.testConfig.adf_aps.host
+            });
 
-            await startProcessPage.selectFromProcessDropdown(processModelWithoutSe);
-            await startProcessPage.deleteDefaultName();
+            await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
 
-            await browser.sleep(1000);
+            const newTenant = await this.alfrescoJsApi.activiti.adminTenantsApi.createTenant(new Tenant());
```

Comment:
can we wrap this API calls inside a try-catch and a message so we can understand that if something fails here is inside the beforeAll?

Reply:
done